src/use_case/settings/RoomsSettingsDataAccessInterface.java

```diff
@@ -0,0 +1,7 @@
+package use_case.settings;
+
+import entities.auth.User;
+
+public interface RoomsSettingsDataAccessInterface {
```

Comment:
Why is this in this PR?

Reply:
You should be able to do this via rooms state only

Reply:
No because we also need an access token